### PR TITLE
Release ipyvuetify 0.1.3 on conda-forge

### PR DESCRIPTION
**Pull request by mariobuikhuizen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by maartenbreddels** [comment]
Happy to be a maintainer

File: recipes/ipyvuetify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
```
Comment (scopatz) [inline recipes/ipyvuetify/meta.yaml]: this needs real tests

File: recipes/ipyvuetify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
```
Comment (maartenbreddels) [inline recipes/ipyvuetify/meta.yaml]: This needs dependencies, i think in setup.py we only have ipywidgets
```suggestion
   - ipywidgets >=7.0
```

**Comment by maartenbreddels** [comment]
Not sure why the tests fail :/

**Comment by maartenbreddels** [comment]
@scopatz is it ok like this, would be great to have this in for scipy.

File: recipes/ipyvuetify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >= 7.0.0
```
Comment (chrisburr) [inline recipes/ipyvuetify/meta.yaml]: FYI: Spaces aren't allowed in version constraints so the correct way of writing this is:
```yaml
    - ipywidgets >=7.0.0
```
(I've fixed it in https://github.com/conda-forge/staged-recipes/commit/0c19fdc6dac8aeeb33b353e46b0a321758a281d4)

**Pull request merged**